docs/setup.rst

```diff
@@ -94,18 +95,97 @@ The following sub-sections provide guidance on how to install the recommended to
 - **Linux** TODO
 - **macOS** TODO
 
+.. _setup_zlib:
+
+zlib
+----
+:zlib:`zlib<>` is a common compression library.
+It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.
+
+- **Windows** On Windows, it's easiest to install using the packaged version available from :opencmiss_repo:`the OpenCMISS site<>` .
+  You're welcome to build your own version if you'd rather, but please make sure it's a 64-bit implementation.
```

Comment:
`You are...` not `You're...` Same for `you'd` which should be replaced with `you would`, etc.

Reply:
e1b924d